apps/auth_zero/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import include
+from django.urls import path
```

Comment:
Codacy found an issue: [No name 'path' in module 'django.urls'](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6329042)